### PR TITLE
[Ahuna] AssistantBuilder doesn't load agentConfigs, uses names route

### PR DESCRIPTION
**Pull request by philipperolet**
Related [task](https://github.com/dust-tt/dust/issues/2970)

Side-effect: allows reuse of already used avatars for agents

### CODE REVIEW

File: front/components/assistant_builder/AssistantBuilder.tsx [hunk 1]
```diff
@@ -84,6 +81,17 @@ const usedModelConfigs = [
   GEMINI_PRO_DEFAULT_MODEL_CONFIG,
 ];
 
+// Avatar URLs
+const BASE_URL = "https://dust.tt/";
+const buildAvatarUrl = (basePath: string, fileName: string) =>
```
Comment (flvndvd) [inline front/components/assistant_builder/AssistantBuilder.tsx:86]: You can use `path` here to format 😛.

**Review by flvndvd — APPROVED** [review]
LGTM

File: front/components/assistant_builder/AssistantBuilder.tsx (hunk 1 above)
Comment (philipperolet) [reply front/components/assistant_builder/AssistantBuilder.tsx:86]: hmm sorry copilot refused :sweat_smile: 
image

Comment (flvndvd) [reply front/components/assistant_builder/AssistantBuilder.tsx:86]: Oh, I meant to say 'URL' in my previous message. No worries, though, it's all good!

**Review by fontanierh — COMMENTED** [review]
This is removing the logic where we first check if the avatar is used though no ? 
Isn't it better to pick a new avatar if not already exhausted ?

**Comment by spolu** [comment]
It does remove it but that logic does not hold for workspace that have consumed all avatars anyway.
 A random choice makes thing much more lightweight and there is low probability of a collision between assistants that are close in names and concepts.

**Comment by fontanierh** [comment]
> It does remove it but that logic does not hold for workspace that have consumed all avatars anyway.
A random choice makes thing much more lightweight and there is low probability of a collision between assistants that are close in names and concepts.

Feels like a bit of a regression for the vast majority of workspaces that have **not** exhausted avatars. But it doesn't matter enough for me to fight for it !

**Comment by spolu** [comment]
Do you believe there is a real negative impact of having a repetition of avatar?

With low number of agents it's unlikely
With large number of agents the impact feels completely minor?

**Comment by fontanierh** [comment]
> Do you believe there is a real negative impact of having a repetition of avatar?

I believe it's not as good of an experience compared to have strictly different avatars. They are supposed to be AI generated images (even if we hack by pre-generating them). Ideally, you would never exhaust them, you would always get a fresh one. IMHO, we should aim for having more avatars than what people could possibly use.

**Comment by fontanierh** [comment]
But again this is pretty minor and if we are fine removing this little nice-to-have then I won't oppose

**Pull request merged**